pages/common/docker-image.md

```diff
@@ -0,0 +1,21 @@
+# docker image
+
+> Manage Docker images.
+> See also `docker build`, `docker import`, and `docker pull`.
+> More information: <https://docs.docker.com/engine/reference/commandline/image/>.
+
+- List local Docker images:
+
+`sudo docker image ls`
+
+- Delete unused local Docker images:
+
+`sudo docker image prune`
+
+- Delete all unused images, not just those without a tag:
```

Comment:
What do you think about this?  
```suggestion
- Delete all unused images (not just those without a tag):
```
The original confused me at first.

Reply:
Better

Reply:
I don't really see a need for the change, but I'll apply it anyway.